addons/binding/org.openhab.binding.km200/OSGI-INF/org.openhab.binding.km200.internal.KM200ChannelTypeProvider.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Given that you used the annotations please remove these xml files and add .gitignore to not add them again, like other bindings do.